app/controllers/training_library_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class TrainingLibraryController < ApplicationController
+  def create
+    training_library = TrainingLibrary.new(training_library_params)
+    if training_library.save
+      render json: { status: 'success', data: training_library }, status: :created
+    else
+      render json: { status: 'error', message: training_library.errors.full_messages },
```

Comment:
why's this called `message` if its an array of messages?

Reply:
i have renamed it to 'errorMessages'